c7n/resources/elb.py

```diff
@@ -202,18 +286,34 @@ def is_ssl(b):
 
 @filters.register('security-group')
 class SecurityGroupFilter(net_filters.SecurityGroupFilter):
+    """ELB security group filter"""
```

Comment:
these should not have a doc string... the base class should have common docs across subclasses.